src/Libraries/CoreNodes/Properties/Resources.resx

```diff
@@ -234,4 +234,16 @@
   <data name="DefineDataSupportedInputValueExceptionMessage" xml:space="preserve">
     <value>Input must be a single value or a non-nested list.</value>
   </data>
-</root>
+  <data name="CurveMapperEqualMinMaxWarning" xml:space="preserve">
+    <value>• Min and Max values must not be the same.</value>
```

Comment:
Helena suggested to change this message to "Min and Max values must be different.". Can you make that change in the other Curvemapper PR